layouts/_default/list.html

```diff
@@ -37,8 +37,12 @@
                                     <h2><a href="{{ .Permalink }}">{{ .Title }}</a></h2>
                                     <div class="clearfix">
                                         <p class="author-category">
-                                          {{ if isset .Params "author" }}
-                                          {{ i18n "authorBy" }} <a href="#">{{ .Params.author }}</a>
+                                          {{ if isset .Params "authors" }}
```

Comment:
there is a space before the comma:

<img width="328" alt="Screenshot 2021-08-03 at 16 18 59" src="https://user-images.githubusercontent.com/1152728/128031966-31360233-6778-47f9-8854-54fea5b94e28.png">

Reply:
Fixed that.